app/instrumentation-tests/src/org/commcare/utils/Array+Extension.kt

```diff
@@ -0,0 +1,15 @@
+package org.commcare.utils
+
+import androidx.test.espresso.matcher.ViewMatchers.withText
+
+fun Array<String>.areDisplayed() {
```

Comment:
Does these make more sense as the Matcher Extension rather than an array extension ? For eg. defining a `isArrayDisplayed(items: array)` as a matcher extension. I think so because the primary purpose of these methods is to match the text in arrays and I would classify these methods to be part of Matcher rather than Array, but would like to know if there is a particular benefit of keeping them as Array extensions ?

Reply:
I don't think `isArrayDisplayed(items: array)` can be used as an extension. It'll be more like a utility method where we'll have to call it `InstrumentationUtiility.isArrayDisplayed(items)`. 
I wanted to make use of extension here merely because of the way it will be used. Like we can simply call `items.areDisplayed()` and that's it. And it is kinda more readable this way.

Reply:
Makes sense. Thanks for the explanation.